lib/ring_logger.ex

```diff
@@ -116,11 +116,27 @@ defmodule RingLogger do
   @spec configure([server_option]) :: :ok
   defdelegate configure(opts), to: Server
 
+  @doc """
+  Update the Log level for the given module
+
+  Note that `:_` will change the default level all modules
+
+  For example, to make MyModule :warn, while defaulting to :debug:
+  iex> RingLogger.level(MyModule, :warn)
+  :ok
+
+  iex> RingLogger.level(:_, :debug)
+  :ok
+  """
+  def level(module, level) do
+    Logger.configure_backend(__MODULE__, module_levels: %{module => level})
+  end
+
   #
   # Logger backend callbacks
   #
   def init(__MODULE__) do
-    {:ok, init({__MODULE__, []})}
+    init({__MODULE__, []})
```

Comment:
This looks like an unrelated bug fix, right?

Reply:
I don't know what was going on there. @michaelkschmidt?

Reply:
That is required to get the state to return correctly.  Otherwise you wind up with a state of {:ok, %{}} instead of %{}

Reply:
"state" here being the module_levels